docs/tutorials/calibrating_armonk.ipynb

```diff
@@ -0,0 +1,1694 @@
+       "      <td>True</td>\n",
+       "      <td>1b39ae30-8515-4906-a575-78c1614af593</td>\n",
+       "      <td>default</td>\n",
+       "      <td>(0,)</td>\n",
+       "      <td>amp</td>\n",
+       "      <td>xp</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>0.500000</td>\n",
+       "      <td>2021-07-16 13:11:30.812487+0000</td>\n",
+       "      <td>True</td>\n",
+       "      <td></td>\n",
+       "      <td>default</td>\n",
+       "      <td>()</td>\n",
+       "      <td>amp</td>\n",
+       "      <td>x90p</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      value                        date_time  valid  \\\n",
+       "0  0.500000  2021-07-16 13:11:30.812427+0000   True   \n",
+       "1  0.780617  2021-07-16 15:13:58.432000+0200   True   \n",
+       "2  0.500000  2021-07-16 13:11:30.812487+0000   True   \n",
+       "\n",
+       "                                 exp_id    group qubits parameter schedule  \n",
+       "0                                        default     ()       amp       xp  \n",
+       "1  1b39ae30-8515-4906-a575-78c1614af593  default   (0,)       amp       xp  \n",
+       "2                                        default     ()       amp     x90p  "
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pd.DataFrame(cals.parameters_table(qubit_list=[qubit, ()], parameters=\"amp\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "about-train",
+   "metadata": {},
+   "source": [
+    "## 4. Calibrating the value of the DRAG coefficient\n",
+    "\n",
+    "A Derivative Removal by Adiabatic Gate (DRAG) pulse is designed to minimize leakage\n",
+    "to a neighbouring transition. It is a standard pulse with an additional derivative\n",
+    "component. It is designed to reduce the frequency spectrum of a normal pulse near\n",
+    "the $|1\\rangle$ - $|2\\rangle$ transition, reducing the chance of leakage\n",
+    "to the $|2\\rangle$ state. The optimal value of the DRAG parameter is chosen to\n",
+    "minimize both leakage and phase errors resulting from the AC Stark shift.\n",
+    "\n",
+    ".. math::\n",
+    "\n",
+    "    f(t) = \\Omega_x(t) + j \\beta \\frac{\\rm d}{{\\rm d }t} \\Omega_x(t)\n",
+    "\n",
+    "Here, $\\Omega_x$ is the envelop of the in-phase component of the pulse and\n",
+    "$\\beta$ is the strength of the quadrature which we refer to as the DRAG\n",
+    "parameter and seek to calibrate in this experiment. \n",
+    "The DRAG calibration will run\n",
+    "several series of circuits. In a given circuit a Rp(β) - Rm(β) block is repeated\n",
+    "$N$ times. Here, Rp is a rotation with a positive angle and Rm is the same rotation\n",
+    "with a native angle. As example the circuit of a single repetition, i.e. $N$, is\n",
```

Comment:
```suggestion
    "with a negative angle. As example the circuit of a single repetition, i.e. $N$, is\n",
```

but "negative angle" isn't quite right -- you should say pi rotated angle or negative amplitude.

Reply:
"single repetition" is a little confusing (because it is a single set of 3 repeats). Maybe change "i.e. $N$" to "i.e. $N=3$".

Reply:
Indeed. Fixed here: 622960d